mod_ngarn/worker.py

```diff
@@ -82,7 +82,7 @@ class JobRunner:
     async def fetch_job(
         self,
         cnx: asyncpg.Connection,
-        table: str = escape_table_name(os.getenv('DBTABLE', 'modngarn_job')),
+        table: str = escape_table_name(os.getenv("DBTABLE", "modngarn_job")),
```

Comment:
I wouldn't keep hard coding this logic. You should abstract this policy out to a single place in the code.